routes/get.js

```diff
@@ -48,7 +48,7 @@ module.exports = (req, res) => {
     gravatar = `<img id="gravatar" alt="Profile image" src="https://www.gravatar.com/avatar/${md5(
       user.email.trim().toLowerCase()
     )}" />`;
-  } else if (typeof user.copyright[0] === 'object' && user.gravatar) {
+  } else if (user.copyright && typeof user.copyright[0] === 'object' && user.gravatar) {
```

Comment:
I thought this could never happen because of the middleware I put in place? Just curious.

Reply:
Well, it actually did happen.

The line change solved the following error when calling the root domain:
```
Uncaught TypeError: Cannot read property '0' of undefined
```